commit 9bd43ab747e6f3cc624258942f023308337d5b6c
Author: Daniel Matuschek <info@crazy-audio.com>
Date:   Thu Jan 16 07:36:35 2014 +0100

    ASoC: wm8804: Set idle_bias_off to false Idle bias has been change to remove warning on driver startup
    
    Signed-off-by: Daniel Matuschek <daniel@matuschek.net>

diff --git a/sound/soc/codecs/wm8804.c b/sound/soc/codecs/wm8804.c
index 9a7a289..32e199a 100644
--- a/sound/soc/codecs/wm8804.c
+++ b/sound/soc/codecs/wm8804.c
@@ -675,7 +675,7 @@ static struct snd_soc_codec_driver soc_codec_dev_wm8804 = {
 	.suspend = wm8804_suspend,
 	.resume = wm8804_resume,
 	.set_bias_level = wm8804_set_bias_level,
-	.idle_bias_off = true,
+	.idle_bias_off = false,
 
 	.controls = wm8804_snd_controls,
 	.num_controls = ARRAY_SIZE(wm8804_snd_controls),
